src/Service/Kinesis/CHANGELOG.md

```diff
@@ -2,10 +2,15 @@
 
 ## NOT RELEASED
 
+## 2.0.0
+
+### BC-BREAK
+
+- BC BREAK: `listStreams` now yield over stream names AND descriptions
```

Comment:
Am I right here? 

For reference, this has been introduced by AWS in https://github.com/async-aws/kinesis/commit/0841292426cadbe269a54f102b750935f77219da